Create (S)CSS-flavored Flame implementation package (#109)
lightspeed/flame: packages/flame-css/README.md

> # Flame css

Reviewer: ```suggestion
# Flame CSS
```

> `flame-css` is an alternative offering to `flame`.

Reviewer: ```suggestion
A pure CSS alternative offering to [Flame React](https://github.com/lightspeed/flame#readme).
```

> It offers a sass codebase and an easy to consume css file.

Reviewer: I would avoid mentioning Sass in the intro. Perhaps later down as an "Unsupported" feature that we might drastically change or remove at some point.

@TODO:

- [ ] Hook flame-tokens/flame-foundations as primary variables

> - [ ] Integrate tailwind to output utility classes

Reviewer: Let's rephrase this to avoid implementation details. Let's instead mention that we'd like to include a guide on how to integrate [utility-first CSS](https://css-tricks.com/in-defense-of-utility-first-css/) on top of Flame CSS.

- [x] Storybook setup
- [x] Write all stories
- [ ] Snapshot testing

> - [ ] Publishing

Reviewer: Publishing will be done as soon as we merge/`yarn release` right?

Author: gonna remove that particular TODO as it was only pertinent for my personal tracking.

## Caveat

> Although we are bundling the sass source code, for the most part, you should only be using the generated css file.

Reviewer: ```suggestion
Although we are bundling the Sass source code,  you should only be using the generated CSS file in the majority of cases.
```

Ah, here's what I mentioned in the README intro. I would move this at the top so that it's clear from the start.

`flame-css` is not meant to be a complete replacement to `flame`, as it lacks many more complex functionalities
that just are not possible without some front-end framework.

However, `flame-css` provides all the necessary basic components and utility functions to build out the complex
functionalities in whatever framework you choose.

> ## Installing

Reviewer: Let's move this above the `caveat` section, and add in a step for [font setup](https://github.com/lightspeed/flame#link-fonts) (I'd copy/paste in case Flame React mainline changes font setup along the way).

> If you are leveraging some form of bundler (webpack, parcel...), you may wish to import the compiled css file

Reviewer: I'd make it clear this is the recommended way, not "if you are leveraging...".

into the root of your app.

> Please ensure you have the appropriate loaders before importing a `.css` file

Reviewer: Link to webpack loaders?
